src/components1/Activity-Feed.tsx

```diff
@@ -0,0 +1,7 @@
+export default function ActivityFeed(){
```

Comment:
The folder should be called components not components1. And I think it would be better if you made a subfolder for each component as you did with Header. So a subfolder ActivityFeed with ActivityFeed.css and ActivityFeed.tsx inside of it, a subfolder Login with Login.css and Login.tsx inside of it and so on...

also convention for react files is without dashes